types/smartystreets-javascript-sdk/smartystreets-javascript-sdk-tests.ts

```diff
@@ -25,6 +26,24 @@ client
         err, // $ExpectType any
     ) => console.error(err));
 
+// usStreet - Batch
+client = SmartyStreetsSDK.core.buildClient.usStreet(credentials);
+batch = new SmartyStreetsSDK.core.Batch<SmartyStreetsSDK.usStreet.Lookup>();
+lookup = new SmartyStreetsSDK.usStreet.Lookup();
+lookup.street = '4646 Scott St';
```

Comment:
Is Frenchy's Chicken that good?

Reply:
[Queen B](https://twitter.com/BeyLegion/status/906316723594387456?s=20) approves... no cap.